.github/workflows/prune-dist.yml

```diff
@@ -28,14 +25,15 @@ jobs:
         run: |
           set -ex
 
-          HEAD=$(git rev-parse HEAD)
+          # head commit SHA of default branch
+          HEAD=$(git ls-remote 'https://github.com/${{ github.repository }}' main master | cut -f1 | head -n1)
```

Comment:
`GITHUB_SERVER_URL`

Reply:
Nice! I did not know that one. Documented at https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
Added as separate commit, as that is not synced from cockpit, and affects many other workflows.